features/features.json

```diff
@@ -1,4 +1,9 @@
 [
+  {
+    "version": 2,
+    "id": "hide-footer",
+    "versionAdded": "v2.30.1"
```

Comment:
This can be updated to `v3.0.0`

Reply:
@StioStudio why did you mark this as resolved?

Reply:
Oh mb the code didn't update for me for a second, sorry